EIPS/eip-draft_Non-Fungible Token with usage rights_abbrev.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Non-Fungible Token with usage rights"
```

Comment:
```suggestion
title: Non-Fungible Token with usage rights
description: Separation of Non-Fungible Token ownership and usage rights , Truly in line with real life.
```
Description is required.  Simple Summary is no longer a required field.  See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for an example to follow.